modules/end-user-guide/partials/ref_devfile-reference.adoc

```diff
@@ -639,7 +639,7 @@ the {platforms-namespace} to which the workspace is deployed.
 * `CHE_MACHINE_AUTH_SIGNATURE__ALGORITHM`: The encryption algorithm used in the secured communication with the {prod-short} server.
 
 
-A devfiles may only need the `CHE_PROJECTS_ROOT` environment variable to locate the cloned projects in the component's container. More advanced devfiles might use the `CHE_WORKSPACE_LOGS_ROOT__DIR` environment variable to read the logs (for example as part of a devfile command). The environment variables used to securely access the {prod-short} server are out of scope for devfiles and are present only for advanced use cases, which are handled by the {prod-short} plug-ins.
+A devfiles may only need the `CHE_PROJECTS_ROOT` environment variable to locate the cloned projects in the component's container. More advanced devfiles might use the `CHE_WORKSPACE_LOGS_ROOT__DIR` environment variable to read the logs (for example as part of a devfile command). The environment variables used to securely access the {prod-short} server are out of scope for devfiles and are available only for advanced use cases, which are handled by the {prod-short} plug-ins.
```

Comment:
`A devfiles`?
may > might
IBM SG:
```
may verb
Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.
"May" is overused in technical information. In most cases, use "can" or "might" instead.
```

Reply:
```suggestion
A devfile might need the `CHE_PROJECTS_ROOT` environment variable to locate the cloned projects in the component's container. More advanced devfiles might use the `CHE_WORKSPACE_LOGS_ROOT__DIR` environment variable to read the logs. The environment variables for securely accessing the {prod-short} server are out of scope for devfiles. These variables are available only for advanced use cases, which are handled by {prod-short} plug-ins.
```

Reply:
IBM SG: Use parentheses to identify items such as abbreviations, symbols, and measurements, but avoid using parentheses in general text.

Reply:
Is `(for example as part of a devfile command)` redundant information?

Reply:
IBM SG: Use simple verbs and tenses. Use simple language. Avoid needless words. Try to keep
sentences to an average of 32 words or fewer, and vary sentence lengths.